.github/workflows/node.js.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       run: >
           git config --global url."https://github.com/".insteadOf
           ssh://git@github.com/
-    - run: npm ci
-    - run: npm run codegen
-    - run: npm run build
-    - run: npm run test
+    - run: yarn
+    - run: yarn codegen
+    - run: yarn build
+    - run: yarn test
```

Comment:
using `yarn` instead of `npm` so the package-lock can be gotten rid off.

Reply:
Why would you want to remove package-lock? The reason we have them both is because some people prefer using npm and some prefer using yarn.